src/Features/Core/Portable/ExtractMethod/MethodExtractor.TriviaResult.cs

```diff
@@ -42,10 +42,11 @@ public async Task<OperationStatus<SemanticDocument>> ApplyAsync(GeneratedCode ge
                 var triviaResolver = GetTriviaResolver(method);
 
                 // Failed to restore the trivia.  Just return whatever best effort result is that we have so far.
-                return annotationResolver == null || triviaResolver == null
-                    ? OperationStatus.Succeeded.With(document)
-                    : OperationStatus.Succeeded.With(
-                        await document.WithSyntaxRootAsync(_result.RestoreTrivia(root, annotationResolver, triviaResolver), cancellationToken).ConfigureAwait(false));
+                if (annotationResolver == null || triviaResolver == null)
+                    return document;
```

Comment:
another case where we no longer have to bail out after trying to do something.  instead, if that happens (which it never should), we can still move forward, just in a slightly suboptimal state.  Specifically, this was a helper that tries to ensure the exact correct leading whitespace before/after the extracted method statements.  If this doesn't work for some reason, we shouldn't block EM entirely.